tests/trading/test_claimTradingProceeds.py

```diff
@@ -220,6 +220,6 @@ def test_reedem_failure(kitchenSinkFixture, cash, market):
         claimTradingProceeds.claimTradingProceeds(market.address, sender = tester.k1)
 
     # set timestamp to 3 days later (waiting period)
-    kitchenSinkFixture.chain.head_state.timestamp += long(timedelta(days = 3, seconds = 1).total_seconds())
+    kitchenSinkFixture.contracts["Time"].incrementTimestamp(long(timedelta(days = 3, seconds = 1).total_seconds()))
```

Comment:
Big fan of this change in all of the python tests.  I wonder if this can simplify some things we had to do in conftest to deal with `time`?  Probably not worth the time to investigate if we are eventually moving off of python....

Reply:
Ill still take a look and see. Might just save it for whenever i get time to work on test speed